pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -138,6 +139,12 @@ func setDefaultConcurrency(local *latest.LocalBuild) {
 	}
 }
 
+func setDefaultToBuildkit(local *latest.LocalBuild) {
+	if !local.UseBuildkit {
+		local.UseBuildkit = true
+	}
```

Comment:
Since `UseBuildkit` is a `bool` and not `*bool`, this effectively always forces `UseBuildkit` to true.

Reply:
no, `nil` case means `UseBuildkit` not set in `skaffold.yaml`, 

whether to fallback to non-buildkit needs to check if buildkit support by docker daemon

https://github.com/GoogleContainerTools/skaffold/blob/b866a05b725956129d0283ab14916337c9e0f034/pkg/skaffold/build/local/docker.go#L49

https://github.com/GoogleContainerTools/skaffold/blob/b866a05b725956129d0283ab14916337c9e0f034/pkg/skaffold/build/local/docker.go#L93